src/unk_020530C8.c

```diff
@@ -315,7 +315,7 @@ void sub_020532A8 (FieldSystem * fieldSystem, BOOL param1)
         sub_02070430(fieldSystem);
     }
 
-    sub_0206B0D4(SaveData_Events(fieldSystem->saveData));
+    Events_ResetVsSeeker(SaveData_Events(fieldSystem->saveData));
```

Comment:
Remind me to come back to this in my PR since we properly label the FieldEvents struct there but I forget if lhea wanted a diff name

Reply:
`VarsFlags` or some variation thereof is what was suggested.